packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -2,212 +2,260 @@
-            type="button"
-            {{on "click" (queue this.closeModal (fn @onRevert this.localDeliveryRule))}}
-          > Cancel
-          </DenaliButton>
-        {{else}}
-          <DenaliButton
-            class="schedule__modal-save-btn m-r-5"
-            disabled={{this.disableSave}}
-            isLoading={{this.isSaving}}
-            {{on "click" (queue (toggle "isSaving" this) this.doSave)}}
-          >
-            {{#if this.localDeliveryRule.isNew}}
-              Save
-            {{else}}
-              Save Changes
-            {{/if}}
-          </DenaliButton>
-          <DenaliButton
-            class="schedule__modal-cancel-btn m-r-5"
-            disabled={{this.isSaving}}
-            style="ghost"
-            type="button"
-            {{on "click" (queue this.closeModal (fn @onRevert this.localDeliveryRule))}}
-          >
-            {{#if this.localDeliveryRule.hasDirtyAttributes}}
-              Cancel
-            {{else}}
-              Close
-            {{/if}}
-          </DenaliButton>
-        {{/if}}
-        {{#unless this.localDeliveryRule.isNew}}
-          <CommonActions::Delete
-            @model={{this.localDeliveryRule}}
-            @deleteAction={{queue this.doDelete this.closeModal}}
-            as |onDelete|
-          >
+      <div class="row flex">
+        <div class="schedule__modal-delivery-list sm-col-1-1 col-2-5 p-x-0">
+          <div class="schedule__modal-new-delivery w-full">
             <DenaliButton
-              disabled={{this.isSaving}}
               @style="text"
-              @size="medium"
-              class="schedule__modal-delete-btn"
-              {{on "click" onDelete}}
+              @size="large"
+              class="w-full p-x-24"
+              {{on "click" this.addNewRule}}
             >
-              Delete
+              <DenaliIcon @icon="add-circle" />
+              New Schedule
             </DenaliButton>
-          </CommonActions::Delete>
-        {{/unless}}
-      {{/if}}
+          </div>
+          <DenaliSidebar class="has-bg-white w-full" as |Sidebar|>
+            {{#each this.deliveryRules as |deliveryRule|}}
+              <Sidebar.Tab
+                  class="schedule__modal-delivery-list-item"
+	                @isActive={{eq this.currentDisplayedRule deliveryRule}}
+	                {{on "click" (fn (mut this.currentDisplayedRule) deliveryRule)}}
+	              >
+                <div class="w-full flex align-items-center">
+                  {{#if deliveryRule.hasDirtyAttributes}}
+                    <div class="schedule__modal-unsaved-rule m-r-10" />
+                  {{/if}}
+                  <div class="schedule__modal-delivery-list-name flex-1">
+                    <p class="is-bold">{{unless deliveryRule.name deliveryRule.defaultName deliveryRule.name}}</p>
+                    {{#if deliveryRule.hasDirtyAttributes}}
+                      <p class="is-italic">&nbsp;unsaved</p>
```

Comment:
```suggestion
                      <p class="is-italic">unsaved</p>
```
Do we need the non breaking space?

Non-breaking space
--


<br class="Apple-interchange-newline">

Non-breaking space
--


<br class="Apple-interchange-newline">